### ISSUE TITLE
Display Axis with Intervals

### ISSUE THREAD

**Issue by codename000** [Live-Charts/Live-Charts#1018]
This is more of a question rather than an issue.
I have created the charts, but am trying to format the axis and its layout, but struggling a bit.

Currently, the chart look something like this 
![image](https://user-images.githubusercontent.com/10926938/69912136-16ca6400-1460-11ea-9c52-0abf06ca8e9f.png)

And this is what I am trying to achieve:
![image](https://user-images.githubusercontent.com/10926938/69912180-86d8ea00-1460-11ea-9601-e2c4a21ddb99.png)
i.e. To have major solid grid lines with value labels on the x axis, then some minor dashed grid lines without any labels.

Much appreciated for any helps at all.

PS. The charts are generated in code, not xaml

**Comment by Notts90** [comment]
I don't believe Live Charts has a concept of major and minor separators. You only have separators, which you can customise. In XAML you could do something like this:

```XAML
<lvc:CartesianChart>
    <lvc:CartesianChart.AxisY>
        <lvc:Axis Title="Y Values"/>
    </lvc:CartesianChart.AxisY>
    <lvc:CartesianChart.AxisX>
        <lvc:Axis Title="X Values">
            <lvc:Axis.Separator>
                <lvc:Separator Stroke="Gray"/>
            </lvc:Axis.Separator>
        </lvc:Axis>
    </lvc:CartesianChart.AxisX>
</lvc:CartesianChart>
```
I believe you can so the same with code as well

**Comment by codename000** [comment]
Thx Notts90. I did exactly that to get around it in the end.
